data/json/npcs/exodii/exodii_merchant_talk.json

```diff
@@ -383,6 +419,248 @@
       { "text": "Actually, I changed my mind.  I have to go.", "topic": "TALK_DONE" }
     ]
   },
+  {
+    "id": "TALK_EXODII_MERCHANT_Experiment_Cyborg_Rescue_Ongoing",
+    "type": "talk_topic",
+    "dynamic_line": { "//~": "Nay, mate, still operating", "str": "Nay, mate, still operating" },
+    "responses": [
+      { "text": "Never mind, I want to ask you something.", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "Ok.  See you later then.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Experiment_Cyborg_Rescue_Complete",
+    "type": "talk_topic",
+    "dynamic_line": { "//~": "Give me a second, and I'll bring them here.", "str": "Give me a second, and I'll bring them here." },
+    "responses": [
+      {
+        "text": "Of course, it is time for us to talk.",
+        "condition": { "u_has_var": "dialogue_exodii_cybor_rescue_ongoing_current_type", "value": "yank_out_all_bionics_from_prototype" },
+        "effect": [
+          { "u_add_var": "dialogue_exodii_cybor_rescue_ongoing", "value": "no" },
+          { "u_add_var": "dialogue_exodii_cybor_rescue_ongoing_current_type", "value": "none" },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 20 } }, "then": { "u_spawn_item": "bio_nanobots" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 4 } }, "then": { "u_spawn_item": "bio_armor_arms" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 6 } }, "then": { "u_spawn_item": "bio_armor_eyes" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 4 } }, "then": { "u_spawn_item": "bio_armor_head" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 4 } }, "then": { "u_spawn_item": "bio_armor_legs" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 3 } }, "then": { "u_spawn_item": "bio_armor_torso" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 10 } }, "then": { "u_spawn_item": "bio_carbon" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 10 } }, "then": { "u_spawn_item": "bio_climate" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 9 } }, "then": { "u_spawn_item": "bio_digestion" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 8 } }, "then": { "u_spawn_item": "bio_flashlight" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 20 } }, "then": { "u_spawn_item": "bio_gills" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 20 } }, "then": { "u_spawn_item": "bio_leukocyte" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 20 } }, "then": { "u_spawn_item": "bio_metabolics" } },
+          { "if": { "x_in_y_chance": { "x": 1, "y": 15 } }, "then": { "u_spawn_item": "bio_synlungs" } },
```

Comment:
All of this *needs* to be using an item group. `u_spawn_item` can accept item groups [with the `use_item_group` argument. ](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/NPCs.md#trade--items)

Reply:
Done cool idea